help/archive-a-channel.md

```diff
@@ -44,6 +42,56 @@ users](/help/manage-user-channel-subscriptions) from it prior to archiving.
 
 {end_tabs}
 
+## View archived channels
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{relative|channel|all}
+
+1. Select **Archived channels** from the dropdown next to the **Filter** box
+   above the list of channels.
+
+{end_tabs}
+
+## Hide content in an archived channel
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{relative|channel|all}
+
+1. Select **Archived channels** from the dropdown next to the **Filter** box
+   above the list of channels.
+
+1. Select a channel.
+
+1. Select the **Subscribers** tab on the right.
+
+1. Remove all subscribers.
+
+{!select-channel-view-general.md!}
+
+1. Under **Channel permissions**, [make the channel
+   private](/help/change-the-privacy-of-a-channel).
+
+    !!! warn ""
+
+        Do the following only if you are an [administrator](/help/user-roles)
+        for your organization. Otherwise, you will lose your ability to administer
+        the channel if you remove your own permissions.
+
+1. Under **Advanced configurations**, remove everyone from **Who can administer
+   this channel**, **Who can subscribe to this channel**, and **Who can
+   subscribe anyone to this channel**. These permissions give users content
+   access to the channel.
```

Comment:
"Who can administer" does not grant content access, just metadata access.

Reply:
Oh, right. Lemme fix.

Reply:
Fixed!

Reply:
Not bothering to update screenshots.